src/AsyncSessionFactory.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Interop\Session\Middleware\Async;
+
+use Interop\Session\Manager\{SessionManagerInterface as SessionManager};
+use Interop\Session\Utils\ArraySession\SavableSession as Session;
+
+class AsyncSessionFactory
```

Comment:
There is an interface missing on this class.
I think we should have a SessionFactoryInterface with a single "get" method to retrieve the session object.

That way, we can inject the factory in any service easily.